### DISCUSSION TITLE
[Developer Tool] Gov4Git: Decentralized governance for open-source communities

### DISCUSSION THREAD

**Discussion by petar** [filecoin-project/community#687] [Project Showcase]
Hi folks!

I am writing on behalf of the [Gov4Git](https://github.com/gov4git/gov4git) project, which is part of the PL network.

We want to invite you to try out our software, essentially take it for a "test drive" and let us know what you think! 

Gov4Git is a protocol for managing open source collaboration on GitHub. It addresses a slew of common frictions and inefficiencies in collaborative open-source projects, by enabling communities to:

- __prioritize issues__ in a transparent and fair community-driven manner,
- __relieve maintainers__ from being a bottleneck in triage,
- objectively __quantify individual contributions__ based on peer review mechanisms, and even
- make __decision-making__ on contentious issues inclusive, open, and democratic.

The relevant links for acquainting yourselves with Gov4Git are:
- [Our website](https://gov4git.org)
- [Overview of the product and how it integrates into a GitHub project](https://www.gov4git.org/s/DEPLOY-MANAGE-COLLABORATE-OVERVIEW.pdf)
- [Step-by-step user's guide](https://github.com/gov4git/gov4git/blob/v2/doc/DEPLOY-MANAGE-COLLABORATE-DETAILED.pdf)
- [Detailed explanation of our governance mechanism for open-source communities](https://medium.com/@gov4git/waimea-collective-management-and-compensation-for-open-source-communities-affa9b9093e9)

We truly thank you for your time and consideration.

**Comment by wolfduan** [comment]
github
